examples/core/core_2d_camera.py

```diff
@@ -14,125 +16,131 @@
     BLUE,
     BLACK,
 )
+# ------------------------------------------------------------------------------------
 
+# ------------------------------------------------------------------------------------
+# Program main entry point
+# ------------------------------------------------------------------------------------
+def main():
+    # Initialization
+    # ----------------------------------------------------------------------------------
```

Comment:
Just out of curiosity, how average user should make that line? Copy it from above?
And then on 3-4 nesting level user must manually cut all excessive dashes?
That convention does not improve anything, just messes with maintainability.